billboard_bronze.cpp

```diff
@@ -0,0 +1,77 @@
+//Jeffrey Chen
+//10 minutes, ****x *****
+//Difficulty - 4/10
+//It didn't seem hard, though I'm not sure why the fifth test case isn't working.
+
+
+
+#include <algorithm>
+#include <iostream>
+#include <sstream> 
+#include <fstream>
+#include <string>
+#include <vector>
+#include <queue>
+#include <stack>
+#include <map>
+#include <set>
+#include <unordered_map>
+#include <unordered_set>
+
+using namespace std;
+
+bool compareTwo(vector<int> arr1, vector<int> arr2) {
+	return arr1[0] < arr2[0];
+}
+
+int main(int argc, char* argv[]) {
+	const char* FIN = "billboard.in";
+	const char* FOUT = "billboard.out";
+	const char* inFile = argc > 1 ? argv[1] : FIN;
+	ifstream cin(inFile);
+	ofstream cout(FOUT);
+
+	int ax1, ay1, ax2, ay2;
+	int cx1, cy1, cx2, cy2;
+	int sum = 0;
+
+	cin >> ax1 >> ay1 >> ax2 >> ay2 >> cx1 >> cy1 >> cx2 >> cy2;
+
+

```

Comment:
Your flaw is in the top-level logic, so I need you to describe it. Write concise but clear comments, like leaving breadcrumbs to yourself, is an art, and an important art for code competition.